### DISCUSSION TITLE
Support mobile browser testing on android and iOS

### DISCUSSION THREAD

**Discussion by RollerIn** [shankybnl/MobileAutomationFramework#20] [Ideas]
Great work, lot of good work is going on 
its great if we add support to test on mobile browsers <on iOS and android>
parallel test execution
auto start Appium server and stop

**Comment by shankybnl** [comment]
Thank you for the suggestions, @RollerIn. 

Appium server auto starts and stops for mac os will be released in the next few days.
I will also work on other suggestions as per demand and the time I get.

**Reply by shankybnl** [reply]
Update - Appium start and stop programmatically is released for Windows and Mac OS. Thank you!

**Reply by EINBOX** [reply]
It's sounds to hear from you, thanks for your time and support
